js/components/operatorSignIn/operatorSelection.tsx

```diff
@@ -48,7 +48,7 @@ export const OperatorSelection = ({
       <Or />
       <label htmlFor={inputId}>Search for an Operator</label>
       <input
-        className="h-[61px]"
+        className="h-[40px]"
```

Comment:
**suggestion:** since 40px is in the [default spacing scale](https://tailwindcss.com/docs/customizing-spacing#default-spacing-scale) could you just do this instead?
```suggestion
        className="h-10"
```

Reply:
ah yes!